src/components/nav/nav-controller.ts

```diff
@@ -1178,7 +1178,12 @@ export class NavController extends Ion {
         transAnimation.duration(0);
       }
 
-      let duration = transAnimation.getDuration();
+      let KEYBOARD_DURATION_PADDING = 0;
+      if ( this._keyboard.isOpen() ){
+        // add 700ms to the click block just to be sure the keyboard is closed
```

Comment:
Remove any specifics about the actual time in the comments, since it'll probably change. so `// add XXms`